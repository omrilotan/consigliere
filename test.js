let cjsInterface;
let esmInterface;

describe("build", () => {
  beforeAll(async () => {
    esmInterface = await import("./index.mjs");
    cjsInterface = require("./index.js");
  });
  test("ESM interface", () => {
    expect(Object.keys(esmInterface)).toMatchSnapshot();
  });
  test("CJS interface", () => {
    expect(Object.keys(cjsInterface)).toMatchSnapshot();
  });
});
