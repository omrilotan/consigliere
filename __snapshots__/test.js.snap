// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`build CJS interface 1`] = `
Array [
  "Logger",
  "NORMALISE",
  "NORMALISE_VALUES",
  "RAW",
  "logger",
]
`;

exports[`build ESM interface 1`] = `
Array [
  "Logger",
  "NORMALISE",
  "NORMALISE_VALUES",
  "RAW",
  "logger",
]
`;
